### ISSUE TITLE
[Bug] Can't work on Jellyfin Andriod app

### ISSUE THREAD

**Issue by kingfxx** [ConfusedPolarBear/intro-skipper#106]
**Describe the bug**
it could work on Windows jellyfin media player and Chrome Web , but can't work in Jellyfin Andriod app -2.4.4 ; When I use Andriod 2.4.4 and open a tv show which could skip the intro on windows client, the skip didn't happen.
**System information**
Official 10.8.7
- Jellyfin installation method: docker
- OS: Synology 7.0 DSM

**Support bundle**

[go to Dashboard -> Plugins -> Intro Skipper -> Support Bundle (at the bottom of the page) and paste the contents of the textbox here]

**Jellyfin logs**

```
paste logs here
```

**Comment by tobi-polar** [comment]
Did you enable autoskip in the plugin settings? afaik the skip button only appears on the web player so if using integrated player youll need auto skip enabled

**Comment by DevilsDesigns** [comment]
yes like i said it works in 10.8.5 same settings same auto skipping and integreated player are all working on web and android and google-tv but once i update to 10.8.7 it broke something to where it doesnt work anymore? I had scanned with 10.8.5 do i need to delete the old scan and rescan all edls again?

**Comment by ConfusedPolarBear** [comment]
1. Can you update your initial post to include your support bundle?
2. Do you have both `Automatically skip intros` and `Automatically skip intros in the first episode of a season` checked in the plugin's settings?
3. Are you using a reverse proxy?
    1. If so, which one?

**Comment by peterguy04** [comment]
Hey 
i believe i am noticing the same issue,  It works fine on my Android TV devices but not my Android phone
 so hopefully i can give more information to this.

**Automatically skip intros** and **Automatically skip intros in the first episode of a season** are both checked

OS: Unraid 6.11.5

Jellyfin: Docker 
lscr.io/linuxserver/jellyfin:latest
Intro Skipper 0.1.7.0
Android application version 2.4.4

Support bundle
* Jellyfin version: 10.8.7
* Plugin version: 0.1.7
* Queue contents: 2957 episodes, 231 seasons
* FFmpeg: `okay`

FFmpeg version:
```
ffmpeg version 5.1.2-Jellyfin Copyright (c) 2000-2022 the FFmpeg developers
built with gcc 9 (Ubuntu 9.4.0-1ubuntu1~20.04.1)
configuration: --prefix=/usr/lib/jellyfin-ffmpeg --target-os=linux --extra-libs=-lfftw3f --extra-version=Jellyfin --disable-doc --disable-ffplay --disable-ptx-compression --disable-shared --disable-libxcb --disable-sdl2 --disable-xlib --enable-lto --enable-gpl --enable-version3 --enable-static --enable-gmp --enable-gnutls --enable-chromaprint --enable-libdrm --enable-libass --enable-libfreetype --enable-libfribidi --enable-libfontconfig --enable-libbluray --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libdav1d --enable-libwebp --enable-libvpx --enable-libx264 --enable-libx265 --enable-libzvbi --enable-libzimg --enable-libfdk-aac --arch=amd64 --enable-libsvtav1 --enable-libshaderc --enable-libplacebo --enable-vulkan --enable-opencl --enable-vaapi --enable-amf --enable-libmfx --enable-ffnvcodec --enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvdec --enable-nvenc
libavutil      57. 28.100 / 57. 28.100
libavcodec     59. 37.100 / 59. 37.100
libavformat    59. 27.100 / 59. 27.100
libavdevice    59.  7.100 / 59.  7.100
libavfilter     8. 44.100 /  8. 44.100
libswscale      6.  7.100 /  6.  7.100
libswresample   4.  7.100 /  4.  7.100
libpostproc    56.  6.100 / 56.  6.100
```

**Comment by kingfxx** [comment]
> 1. Can you update your initial post to include your support bundle?
> 2. Do you have both `Automatically skip intros` and `Automatically skip intros in the first episode of a season` checked in the plugin's settings?
> 3. Are you using a reverse proxy?
>    
>    1. If so, which one?

Thanks for your replay; I have checked these two box both in the setting page.
